.github/workflows/build-test.yml

```diff
@@ -36,25 +36,25 @@ jobs:
 
        - name: Build sources
          run: |
-            $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-strict build-all out-of-tree build-shared
+            sudo $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-strict build-all out-of-tree build-shared test-add-user
 
        - name: Build debian package
          run: |
-            $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh package
+            sudo $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh package
 
-       - name: Run timeless tests
+       - name: Run all tests, excluding timing_tests
          run: |
            cd $GITHUB_WORKSPACE/build
-           make all_tests
+           sudo make all_tests
 
        - name: Run timing tests
          run: |
            cd $GITHUB_WORKSPACE/build
-           make timing_tests
+           sudo make timing_tests
```

Comment:
Will this also include the timing tests in the coverage?

Reply:
Currently not because the `TIMING_TEST` macro is used in module -and integrationtests which would distort the coverage value for unittests if we add this here now.
Possible solution could be to separate the Timing_tests into different targets for module -and integrationtests.

Reply:
Ok i split up the timing_test targets and include them in the coverage report. We need still to have the separation of timing_tests otherwise we will have trouble on MacOS CI build.

Reply:
Should we have to use a different cmd to build and run the test locally on ubuntu or we can keep the current one ?

./tools/iceoryx_build_test.sh clean build-strict debug test

Reply:
You can keep the current one, this executes the tests with scope `all`